index.html

```diff
@@ -289,6 +289,36 @@ <h2 class="header-text">About <br>Myself</h2>
       </div>
     </div>
   </section>
+  <section id="Contact" class="contact-me">
+    <h2 class="intro">Contact me</h2>
+    <p class="intro-contact">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it. </p>
+    <form action="https://formspree.io/f/mlekbpkb" method="POST">
+      <ul>
+        <li>
+          <label for="name">
+            <input class="input" type="text" maxlength="30" placeholder="Abdel-Khafid SALAOU" id="name" name="user_name"  required/>
+          </label>
+        </li>
+        <li>
+          <label for="mail">
+            <input class="input" type="email"  placeholder="khadaf3@gmail.com" id="mail" name="user_email" required/>
+          </label>
+        </li>
+        <li>
+          <label for="msg">
+            <textarea id="msg" name="user_message" maxlength="500" required placeholder="Write your message"></textarea>
+          </label>
+        </li>
+        <li>
+          <label for="button" class="getInTouch">
+            <input type="submit" id="button" name="submit" value="Get in touch" />
+          </label>
+        </li>
```

Comment:
- [ ] With refrence to the figma design, kindly, adjust the styling of your button, to look same as that given below ;:
## Your design:
![image](https://user-images.githubusercontent.com/50990925/220591305-bf3da3b9-df05-4546-8688-45060d72aaeb.png)

## Figma:
![image](https://user-images.githubusercontent.com/50990925/220591464-85cecf53-55b8-412b-8319-62d34925ca32.png)